client/galaxy/scripts/mvc/ui/ui-misc.js

```diff
@@ -10,257 +10,141 @@ define(['utils/utils',
     'mvc/ui/ui-modal'],
     function( Utils, Select, Slider, Options, Drilldown, Buttons, Modal ) {
 
-    /** Image wrapper */
-    var Image = Backbone.View.extend({
-        initialize : function(options) {
-            this.options = Utils.merge(options, {
-                url  : '',
-                cls  : ''
-            });
-            this.setElement(this._template(this.options));
-        },
-        _template: function(options) {
-            return '<img class="ui-image ' + options.cls + '" src="' + options.url + '"/>';
-        }
-    });
-
     /** Label wrapper */
     var Label = Backbone.View.extend({
-        initialize : function(options) {
-            this.options = Utils.merge(options, {
-                title   : '',
-                cls     : '',
-                tagname : 'label'
-            });
-            this.setElement(this._template(this.options));
+        tagName: 'label',
+        initialize: function( options ) {
+            this.model = options && options.model || new Backbone.Model( options );
+            this.tagName = options.tagName || this.tagName;
+            this.setElement( $( '<' + this.tagName + '/>' ) );
```

Comment:
You don't need the two above. 
    
    v1 = Backbone.View({ tagName: 'label' }); 
    v2 = Backbone.View({ tagName: 'li' }); 

will produce

    v1.tagName === 'label'; // and v1.el will be <label></label>;
    v2.tagName === 'li'; // and <li></li>